docs/source/api.rst

```diff
@@ -1,9 +1,71 @@
 API
 ===
 
-.. currentmodule:: ucx_py
+.. currentmodule:: ucp
+
+**ucp**
 
 .. autosummary::
+
```

Comment:
Should we have a blank line like this for other directives or should we drop this one? Only curious if this impacts whether the docs build correctly or not.

Reply:
it doesn't matter but I'll fix to be consistent.  Thanks

Reply:
Ok sorry for the nit then.